### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Resize: Unable to see complete information when we change text size to 200%.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2673]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev1; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Resize; #WCAG1.4.4; #NVDA; #Win10;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Change the display resolution to 1280x 786 and change the text size to 200%
2. Open Visual studio code
3.  Tab till extension button and press enter on it.
4. Search for "Azure Functions" and select it.
5. Tab till "Install" button and press enter on it.
6. Tab till "Azure" button from left side pane and press enter on it.
7. Default screen will get open.
8. Verify everything is visible or not.

### Actual Result:
Unable to see  complete information when we change text size to 200%.

### Expected Result:
User should be able to see complete information when we change text size to till 200%.

### User Impact:
Low vision users find it difficulty in viewing the content and performing operations if few controls and content is not available when we change the text size

### Attachment for Reference:

https://user-images.githubusercontent.com/38523250/108177559-6b233780-7129-11eb-8223-81c1215a7c7c.mp4

[2673_Resize issue_Default screen.zip](https://github.com/microsoft/vscode-azurefunctions/files/6020044/2673_Resize.issue_Default.screen.zip)

**Comment by ejizba** [comment]
Our extension does not control zooming of the UI. Tracked in VS Code by https://github.com/microsoft/vscode/issues/115878
